src/main/scala/org/zalando/nakadi/client/KlientImpl.scala

```diff
@@ -1,11 +1,9 @@
 package org.zalando.nakadi.client
 
 import java.net.URI
-import javax.net.ssl.SSLContext
 
 import akka.actor._
 import akka.http.scaladsl.model.MediaTypes.`application/json`
-import akka.http.scaladsl.{HttpsContext, Http}
```

Comment:
Removed some of the unused imports, but that's not really the intention of this PR. Ignore this. :)